frontend/src/libs/components/checkbox/common.module.scss

```diff
@@ -0,0 +1,67 @@
+@use "sass:color";
+
+@import "src/assets/css/styles.scss";
+
+.container {
+  display: flex;
+  flex-direction: column;
+  font-family: $font-family;
+}
+
+.error {
+  color: $red-dark;
+  font-weight: $font-weight-regular;
+  font-size: 12px;
+  line-height: calc(15.12 / 12);
```

Comment:
Do we need calc here ?

Reply:
of course, It explains what is this value, and how i've obtained it.
![изображение](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/81959259/6a758fbd-8a9c-42db-958b-159b3b610487)

I think line-height: calc(15.12 / 12) is much more self-explanatory than line-height: 1.26, because the next question could be "why 1.26?"